mvcc/backend/backend.go

```diff
@@ -163,6 +165,22 @@ func (b *backend) ForceCommit() {
 }
 
 func (b *backend) Snapshot() Snapshot {
+	stopc, donec := make(chan struct{}), make(chan struct{})
+	go func() {
+		defer close(donec)
+		start := time.Now()
+		ticker := time.NewTicker(snapshotWarningTimeout)
```

Comment:
`defer ticker.Stop()` is more future proof, although there's no bug here at the moment.